instrumentation/opentelemetry-instrumentation-sklearn/src/opentelemetry/instrumentation/sklearn/__init__.py

```diff
@@ -0,0 +1,257 @@
+import logging
+from functools import wraps
+from typing import Callable, Dict, List, MutableMapping, Sequence, Type
+
+from sklearn.base import BaseEstimator
+from sklearn.pipeline import FeatureUnion, Pipeline
+
+from opentelemetry.instrumentation.sklearn.version import __version__
+from opentelemetry.trace import get_tracer
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Can you add a code example in how to instrument?

Reply:
There is a code example in the docstring of `SklearnInstrumentor`. I can move it to the top of the module if that's a better location for it.

Reply:
Yes that would be great. Just to maintain consistency and ease of finding it.

Reply:
Added code examples to module docstring